main/src/ca/uwaterloo/flix/tools/Packager.scala

```diff
@@ -167,20 +167,19 @@ object Packager {
     flix.check() match {
       case Validation.Success(_) => ()
       case Validation.Failure(errors) =>
-        errors.foreach(e => println(e.message.fmt))
+        errors.foreach(e => println(e.message(flix.getFormatter)))
     }
   }
 
   /**
     * Builds (compiles) the source files for the given project path `p`.
     */
-  def build(p: Path, o: Options, loadClasses: Boolean = true)(implicit tc: TerminalContext): Option[CompilationResult] = {
+  def build(p: Path, o: Options, flix: Flix = new Flix(), loadClasses: Boolean = true): Option[CompilationResult] = {
```

Comment:
The flix argument should always go last and typically as an implicit parameter.

Where are you getting this object from?

Reply:
It's configured in Main